libs/aws/langchain_aws/function_calling.py

```diff
@@ -63,6 +63,17 @@
 <description>{parameter_description}</description>
 </parameter>"""
 
+class CachePointType(TypedDict):
+    """Configuration for prompt caching in Bedrock toolConfig."""
+    
+    type: Literal["default"]
+
+
+class CachePoint(TypedDict):
+    """Configuration for prompt caching in Bedrock."""
+    
+    cachePoint: CachePointType
```

Comment:
This might be an overkill for such a small input structure. 
```suggestion
class CachePointType(TypedDict):
    """Configuration for prompt caching in Bedrock toolConfig."""
    
    type: Literal["default"]


class CachePoint(TypedDict):
    """Configuration for prompt caching in Bedrock."""
    
    cachePoint: CachePointType
```